### PR TITLE
Added Awareness to Basic Rules

### PR DESCRIPTION
**Pull request by Reveraine**

### CODE REVIEW

**Review by bleehu — APPROVED** [review]
Shipit!

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -555,13 +555,15 @@ negative effects.
 
 - Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
 - Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
-- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another).
+- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
+- Awareness: passive knowledge and understanding of surroundings (Ex: realizing someone has walked up behind you)
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt]: I just don't like using the word passive here, in player documentation, for some reason. Comprehension of ones surroundings does require some active and critical thinking / processing, even if its no enough for us to represent it mechanically. Probably not a merge blocker.

**Review by Turtlelord26 — APPROVED** [review]
Good initiative 👍

File: 01_01_Basic_Rules.txt (hunk 1 above)
Comment (Reveraine) [reply 01_01_Basic_Rules.txt]: I think the description will work just as well without it there.

**Pull request merged**